djangocms_concurrent_users/templates/djangocms_concurrent_users/concurrency_button.html

```diff
@@ -19,3 +19,11 @@
 
 <link rel="stylesheet" href="{% static 'css/concurrent_users.css' %}">
 
+<div class="cms js-concurrentWarning concurrentWarning">
+    <div class="js-concurrentWarningMessage"></div>
+    <br>
+    <div class="cms-modal-item-buttons">
+        <a href="javascript:window.history.back();" type="button" class="cms-btn js-concurrentWarningButtonBack"></a>
```

Comment:
Why do you use JS here? It is overridden anyway, isn't it? Or is it supposed to be some kind of fallback?